todo_app/data/update_item_demo.py

```diff
@@ -0,0 +1,18 @@
+import requests
```

Comment:
These files are really useful for testing, but once you've used them to get the main app working they tend to get ignored and then drift out of date, so we're best off just deleting them as soon as we're done with them 😄